examples/can_simple.py

```diff
@@ -1,21 +1,29 @@
 """
-Minimal example for controlling an ODrive via the CANSimple protocol.
+Example for controlling an ODrive via the CANSimple protocol with watchdog handling.
 
-Puts the ODrive into closed loop control mode, sends a velocity setpoint of 1.0
-and then prints the encoder feedback.
+Puts the ODrive into closed loop control mode, sends periodic sinusoidal velocity
+setpoints, and asynchronously prints the encoder feedback. The regular velocity
+setpoint update resets the watchdog, which is a safety feature that disengages
+the axis if no valid commands are received within the configured timeout period,
+avoiding uncontrolled behavior.
```

Comment:
The way I think about it, watchdog handling is just a side effect of continuous setpoints. There's nothing that's explicitly for watchdog handling and it works just as well with the watchdog disabled. So we don't have to mention it in the headline and most of the description can stay as it was (except that it's now a sine function instead of constant vel).

You can mention the watchdog as a side thought in just a small paragraph, such as:

> If the watchdog is enabled on the ODrive, it is fed implicitly by the continuous velocity setpoint message and the motor will stop when the script is terminated. The heartbeat interval should be shorter than the watchdog timeout to ensure timely confirmation of the axis entering closed loop control mode without triggering the watchdog.